src/components/CitySlider/CitySlider.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Slider from 'react-slick';
+import OneSlide from '../OneSlide/OneSlide';
+import './CitySlider.css';
+import '../../../node_modules/slick-carousel/slick/slick.css';
+import '../../../node_modules/slick-carousel/slick/slick-theme.css';
```

Comment:
Oh, this is interesting!  What happens when you have a `.gitignore` file that includes all of your `node_modules`?

I'm curious as to how `CitySlider.js` can still access these files!